src/components/status/Status.js

```diff
@@ -9,7 +9,6 @@ export default function Status(props) {
     messages.push(<p className="details" key="info">{info}</p>);
   }
   if (warn) {
-    console.warn(warn);
```

Comment:
This may have changed, but originally, there might not be a pane configured for the messages to show, and we wouldn't want it to be silent. Is the `warn` too much noise now?

Reply:
The warnings are still printed, I just moved the warn statement https://github.com/hubmapconsortium/vitessce/blob/f00e8d2e86567fe2ab797b8bcbd39d97b83256aa/src/components/sourcepublisher/SourcePublisher.js#L18